Signum.Engine/Database.cs

```diff
@@ -22,7 +22,7 @@ namespace Signum.Engine
     public static class Database
     {
         #region Save
-        public static void SaveList<T>(this IEnumerable<T> entities)
+        public static IEnumerable<T> SaveList<T>(this IEnumerable<T> entities)

```

Comment:
If the IEnumerable<T> is a general IEnumerable evaluated lazily, not a list, this code will be error prone:

var invoices = Database.Query<UserEntity>().Select(u => new InvoiceEntity { User = u}).SaveList();

The user expects that you have a list of saved invoices, but the query will be made again. 

To avoid this issue make ToList internally, and return a List<T>